pkg/plugins/secret/secrets_manager/secrets_manager.go

```diff
@@ -113,11 +136,22 @@ func (s *secretsManagerSecretService) Put(sec *secret.Secret, val []byte) (*secr
 }
 
 func (s *secretsManagerSecretService) Access(sv *secret.SecretVersion) (*secret.SecretAccessResponse, error) {
+	newErr := errors.ErrorsWithScope("SecretsManagerSecretService.Access")
+
 	if len(sv.Secret.Name) == 0 {
```

Comment:
Just a comment here. I think I have a personal preference for using s != "" for empty string checks and its more descriptive of the type being checked. The length check can be applied to other elements such as slices and maps